mon-pix/tests/integration/components/feedback-panel-test.js

```diff
@@ -47,41 +38,36 @@ describe('Integration | Component | feedback-panel', function() {
 
   describe('Default rendering', function() {
 
-    it('should display the feedback Panel', function() {
-      // when
-      this.render(hbs`{{feedback-panel}}`);
-      // then
-      expect(this.$()).to.have.lengthOf(1);
-      expectLinkViewToBeVisible(this);
-    });
-
-  });
-
-  describe('Link view (available only when form is closed by default)', function() {
-
     beforeEach(function() {
       this.render(hbs`{{feedback-panel}}`);
     });
 
-    it('should display only the "link" view', function() {
-      expectLinkViewToBeVisible(this);
-    });
-
-    it('the link label should be "Signaler un problème"', function() {
-      expect(this.$(OPEN_LINK).text()).to.contain('Signaler un problème');
+    it('display the feedback panel', function() {
+      expect(this.$()).to.have.lengthOf(1);
+      expect(this.$('.feedback-panel__view--link')).to.have.lengthOf(1);
     });
 
-    it('clicking on the open link should hide the "link" view and display the "form" view', function() {
+    it('clicking on the toggle link should toggle the "form" view', function() {
```

Comment:
`it('should toggle the "form" view when clicking on the toggle link', function() {`